app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if command_list[0] == "cp" and len(command_list) == 3:
```

Comment:
what if we pass empty command?

Reply:
The condition does not mention an empty command